### DISCUSSION TITLE
Add optional clear button when the select is open,  to clear all items

### DISCUSSION THREAD

**Discussion by acetinick** [orchidjs/tom-select#34] [Ideas]
**Is your feature request related to a problem? Please describe.**
Looking for obvious way to clear all items of the select whilst hovering or select box is open.

**Describe the solution you'd like**

<img width="935" alt="Screen Shot 2021-02-24 at 12 54 40 PM" src="https://user-images.githubusercontent.com/571649/108990405-7d5c2300-769f-11eb-8300-f7d62d300536.png">

Add 'times' icon to bottom right, or right of the select box to allow user to clear all selections.  Should be able to be done on hover and whilst the select is open

**Describe alternatives you've considered**
- optional flag to show this, maybe "showClear"
- should show on hover, and whilst its open

**Comment by oyejorge** [comment]
Just pushed out v1.3 with [the clear_button plugin](https://tom-select.js.org/plugins/clear-button/)
